GitOps.md

```diff
@@ -0,0 +1,17 @@
+---
+title: GitOps
+status: Feedback Appreciated
+category: concept
+---
+
+
+## What it is
+
+GitOps is a [set of principles](https://github.com/open-gitops/documents/blob/main/PRINCIPLES.md) for operating and managing software systems. In GitOps, environment specifics are defined, or "declared," in a repository with the application code — the system of record.
+
+## Problem it addresses
+
+Configuration management via YAML or JSON files often becomes unwieldy at scale. These files can grow exponentially, especially in large organizations with numerous developers and frequent code changes. Potential consequences include frequently failed deployments (e.g., usage of the incorrect config files), long and slow disaster recovery from system outages, lack of visibility into system workflows, reduced developer productivity, among other challenges.
```

Comment:
I think we've missed something here. GitOps doesn't solve for the problem of storing config in JSON or YAML. I agree that GitOps solves for, or at least improves DR, but many of the other problems and their root cause suggested in this section aren't addressed by a GitOps workflow.